### PR TITLE
High-Low number game added 

### PR DESCRIPTION
**Pull request by sakshichahal53**
Thanks for contributing. Your contribution might help someone understand the concepts better.

However, to make sure things are working properly, here's a checklist. Just make sure you check them.
- [x] I am reading these checkboxes. 
- [x] Read [contribution guidelines](/docs/Contributing.md)
- [x] Screenshots

Screenshot from 2019-04-27 14-03-45
Screenshot from 2019-04-27 14-03-49

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -410,6 +410,12 @@ $(document).ready(function() {
                     'url': 'https://fast.com/'
                 });
               }
+              else if(data.result.metadata.intentName=="numbergame")
+              {
+                chrome.tabs.create({
+                    'url': '.'
```
Comment (gauthamzz) [inline background.js]: this doesnt look right

**Review by gauthamzz — COMMENTED** [review]
Can you tell me the logic behind this, why does `.` map towards hilogame.html

File: background.js (hunk 1 above)
Comment (sakshichahal53) [reply background.js]: On it. will mend the code.

**Comment by sakshichahal53** [comment]
> Can you tell me the logic behind this, why does `.` map towards hilogame.html

It's a bug there. The url should be pointing to the html file for the game.

**Comment by gauthamzz** [comment]
LGTM. i would prefer to have a page for all text based stuff in a page.

**Pull request merged**